modules/e2ee/OlmAdapter.js

```diff
@@ -155,6 +173,17 @@ export class OlmAdapter extends Listenable {
         return this._keyIndex;
     }
 
+    /**
+     * Updates the current participant key.
+     * @param {Uint8Array|boolean} key - The new key.
+     * @returns {number}
+    */
+    async updateCurrentKey(key) {
```

Comment:
This is not async, can you please remove it while you're here? Also check the use.